### ISSUE TITLE
Behaviour of Chunk.scanLeft does not match standard collections

### ISSUE THREAD

**Issue by culturedsys** [functional-streams-for-scala/fs2#1538]
The behaviour of the scanLeft method in Chunk does not match the behaviour of the method in the standard collections - specifically, the Chunk implementation does not include the initial value, and includes the final value twice, e.g:

```scala
Chunk(List(1, 2, 3)).scanLeft(0)(_ + _).toList == List(1, 3, 6, 10, 10)
```

Whereas

```scala
List(1, 2, 3).scanLeft(0)(_ + _) == List(0, 1, 3, 6, 10)
```

I'm not sure if this is intended or not. If it is intended, it seems like it would be good to mention it  in the documentation. If it isn't intended, there appears to be a simple fix - I can submit a PR.

**Comment by mpilquist** [comment]
@culturedsys Yeah, this looks broken to me. A PR to fix would be great.

**Comment by mpilquist** [comment]
Fixed by #1544
